pkg/helm/v3/converter.go

```diff
@@ -0,0 +1,64 @@
+package v3
+
+import (
+	"github.com/helm/helm-2to3/pkg/common"
+	helm2to3v2 "github.com/helm/helm-2to3/pkg/v2"
```

Comment:
nit: lots of numbers in the package name, hard to make out which one it is in code, maybe better is something like "convertv2" or just "helm2"?